native_words.asm

```diff
@@ -10063,52 +9829,36 @@ xt_to:
                 ora state+1
                 beq _interpret
 
-                ; Well, we're compiling. We want to end up with simple
-                ; code that just takes the number that is TOS and saves
-                ; it in the address of the xt we were just given. So we
-                ; want to compile this routine:
-                ;
-                ;       lda 0,x                 - B5 00
-                ;       sta <ADDR_LSB>          - 8D LSB MSB
-                ;       lda 1,x                 - B5 01
-                ;       sta <ADDR_LSB>          - 8D LSB MSB
-                ;       inx                     - E8
-                ;       inx                     - E8
-                ;
-                ; which at least is nice and short. Other than that, we pretty
-                ; much have to do this the hard and long way, because with the
-                ; LSBs and MSBs, we can't really put the numbers in a data
-                ; range and store them with a loop. Sigh.
+                ; compile a fragment that takes number TOS
+                ; and saves to the address of the xt we were just given.
+                ; we'll use a template and replace marker $ff bytes
+                ; with the appropriate addresses.
```

Comment:
i simplified this a bit with a template but would it be better (simpler, shorter, tho slightly slower) to just generate `LITERAL <tmp1> !` and skip this specialization?

Reply:
this does work out much simpler/shorter